documentation/blog/2023-06-22-svelte-dev-overhaul.md

```diff
@@ -0,0 +1,96 @@
+
+Meet the new svelte.dev, a complete overhaul of the old website.
+
+## Multi-page docs
+
+The huge single-pager was a huge nuisance, in terms of finding stuff(Hullo there Ctrl+F), as well as just having low performance due to the sheer size of the page.
+
+We heard you! The docs are all split up into multiple pages now. And all pages are split into their sections in the sidebar. Wanna know how to use the `class:` directive, or wanna look up the syntax for `{#await}`? All these are just one click away under the `Template Syntax` heading in the sidebar.
+
+All modules exposed by svelte are also listed in the sidebar, under the `Runtime` section.
+
+- [svelte](/docs/svelte)
+- [svelte/store](/docs/svelte-store)
+- [svelte/motion](/docs/svelte-motion)
+- [svelte/transition](/docs/svelte-transition)
+- [svelte/animate](/docs/svelte-animate)
+- [svelte/easing](/docs/svelte-easing)
+- [svelte/action](/docs/svelte-action)
+
+And worry not, all the links from the older website will be redirected to the correct page.
+
+## Search
+
+One thing that the previous single pager website was good at was finding things without any JS. It's just a single html page, just hit <kbr>Ctrl+F</kbr> and search what you're looking for! But now that the site uses different pages, that is not an option anymore
+
+And for that, the new website comes with a search bar, which searches through the docs and the API surface. Hit Ctrl+K(or CMD+F for Mac users) and start searching!
+
+## Lights, TypeScript, Action!
+
+The new website comes with a JavaScript/TypeScript toggle, so you can view the docs in your preferred language. For every module, it's exported types are listed at the bottom of the page for easy reference. The types are automatically generated from Svelte's source code, so they're always up to date.
+
+All the JavaScript and TypeScript code snippet have type hints available right there. Just hover over the variable to see it's type. This allows the docs to be type checked at build time, which makes sure we're never out of date.
+
+We also (finally!) added documentation for [Actions](/docs/svelte-action). Svelte Actions are a way to interact with the DOM, and are a great way to add interactivity to your app. The docs for Actions are also available in TypeScript.
+
+```svelte
+<script lang="ts">
+	import type { Action } from 'svelte/action';
+
+	const foo: Action = (node) => {
+		// the node has been mounted in the DOM
+
+		return {
+			destroy() {
+				// the node has been removed from the DOM
+			}
+		};
+	};
+</script>
+
+<div use:foo />
+```
+
+## Dark mode
+
+After many years of users asking for dark mode on the website so they can read the docs for their night-time coding sessions, we finally added it! The website now has a dark mode toggle, which is also synced with your OS's dark mode settings. It can be toggled from the top navbar(bottom navbar on mobile).
```

Comment:
```suggestion
After many years of users asking for dark mode on the website so they can read the docs for their night-time coding sessions, we finally added it! The website now has a dark mode toggle, which is also synced with your OS's dark mode settings. It can be toggled from the top navbar (bottom navbar on mobile).
```